spec/requests/home_controller_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe 'Home', type: :request do
+  describe 'GET /' do
+    it 'renders the index template' do
+      get '/'
+      expect(response).to render_template(:index)
+    end
+
+    it 'has a 200 status code' do
+      get '/'
+      expect(response).to have_http_status(200)
+    end
+
+    it 'includes correct placeholder text' do
+      get '/'
+      expect(response.body).to include('Welcome to My Blog')
+    end
+  end
+end
```

Comment:
- [ ] Kindly write requests requests tests for all actions in the Posts controller. Writing requests tests for actions guarantees that all requests for that action work as intended. 👍